### PR TITLE
youtube clone with html and css (non-responsive)

### PR DESCRIPTION
**Pull request by saintaze**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,36 +1,588 @@
...
-					<i class="material-icons">message</i>
-					<i class="material-icons" >notifications</i>
-					<span class="bell-icon-badge">9+</span>
-					<i class="material-icons account-icon">account_circle</i>
-			</div>
-		</nav>	
+  <nav class="nav">
+    <div class="menu">
+      <i class="material-icons">menu</i>
+      <img class="youlogo" src="/assets/youlogo.png" />
```
Comment (maelfosso) [inline index.html]: The logo doesn't appear. Use a relative path that is `./assets/youlogo.png`

**Comment by maelfosso** [comment]
Hello,

Thank you for your improvement.

Please, the next time, don't create a new pull request but just use the first one until the modifications were approved.

- [x] **README**: Please update the readme. The author's name must be a link to the GitHub account of authors

- [x] **Images**: The images don't appear. Correct them, please

- [x] **Stickler**: Please correct the `stickler` errors

Thanks.

**Comment by saintaze** [comment]
Hey,

The reason why you see broken images is that you are directly opening HTML page in the browser without serving it from a server but once you greenlight the project we will host on GitHub pages and everything will work fine.

About stickler some of the errors are too strict and unnecessary and don't reflect real life development I have asked other students and seniors in Microverse and they say ignore them so,

**Comment by maelfosso** [comment]
# Good Job

Please, the next time don't use tools to serve the App.
And you can use githack.com to get a live version of your app.

You can submit the project completion form.

**Pull request merged**